### PR TITLE
adds new rules about  the robot placement technical challenge

### PR DESCRIPTION
**Pull request by fabiooju**
This is the new rules from robot placement technical challenge, including the procedure, evaluation, socring and tiebreak criteria. According to the league's objectives, this technical challenge will motivate the automatic positioning of the robots, aiming for compliance with the regular league.

### CODE REVIEW

File: 2024/2024-ssl-el-robot-placement.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+In total, there will be 2 TC's evaluating the Technical Challenge, one to operate the Game Controller and another to keep the points and position the robots. Furthermore, the TC will mark the robot position at the field with tape or chalk for each scenario. 
+
+====Scoring 
+
+For every scenario, the following criteria will be evaluated: 
+
+1. Moving the robot to the desired position (each robot) — +5 
+2. Colliding with a robot during the placement — -1 (just one robot loses points) 
+3. Colliding with obstacles during the placement — -1
```
Comment (JoaoVictor0508) [inline 2024/2024-ssl-el-robot-placement.adoc]: Just to be clear, "obstacles" are the "opponent robots" that will be placed in the scenarios, right?

File: 2024/2024-ssl-el-robot-placement.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+In total, there will be 5 scenarios in this challenge. The order of the scenarios will be in ascending order (scenarios 1 to 5), where all teams are going to do their 3 tries at scenario 1 first, and so on. 
+
+In total, there will be 2 TC's evaluating the Technical Challenge, one to operate the Game Controller and another to keep the points and position the robots. Furthermore, the TC will mark the robot position at the field with tape or chalk for each scenario. 
+
+====Scoring 
+
+For every scenario, the following criteria will be evaluated: 
+
+1. Moving the robot to the desired position (each robot) — +5 
+2. Colliding with a robot during the placement — -1 (just one robot loses points)
```
Comment (JoaoVictor0508) [inline 2024/2024-ssl-el-robot-placement.adoc]:
```suggestion
2. Colliding with a team robot during the placement — -1

```

File: 2024/2024-ssl-el-robot-placement.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+====Scoring 
+
+For every scenario, the following criteria will be evaluated: 
+
+1. Moving the robot to the desired position (each robot) — +5 
+2. Colliding with a robot during the placement — -1 (just one robot loses points) 
+3. Colliding with obstacles during the placement — -1 
+4. Colliding with one of field boundary — -1 
+
```
Comment (JoaoVictor0508) [inline 2024/2024-ssl-el-robot-placement.adoc:40]:
```suggestion

NOTE: A collision between two or more team robots will count as one collision, i.e., the collision will cost -1 for the team, not -1 for each robot.
```

File: 2024/2024-ssl-el-robot-placement.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+3. The team with the most points at scenario 4 wins [difficult scenario]; 
+4. The team with the most points at scenario 3 wins [medium scenario]; 
+5. The team with the most points at scenario 2 wins [easy scenario]; 
+
+Example: team A and B with 10 points; Team A: 4 points at scenario 5 + 6 points at scenario 3; Team B: 6 points at scenario 5 + 4 points at scenario 3; Team B wins. 
+
+==== Scenarios 
+
+1. Kick-Off 
+2. Free Kick
```
Comment (JoaoVictor0508) [inline 2024/2024-ssl-el-robot-placement.adoc]: Are we going to consider the ball in the scenarios too? If so, the ball will be considered as an obstacle for the Scoring?

File: 2024/2024-ssl-el-robot-placement.adoc [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+3. The team with the most points at scenario 3 wins [difficult scenario]; 
+4. The team with the most points at scenario 2 wins [medium scenario]; 
+5. The team with the most points at scenario 1 wins [easy scenario]; 
+
+Example: team A and B with 10 points; Team A: 4 points at scenario 4 + 6 points at scenario 3; Team B: 6 points at scenario 4 + 4 points at scenario 3; Team B wins. 
+
+==== Scenarios 
+
+=== Scenario 1: Free Kick 1
+== Start
```
Comment (ersaraujo) [inline 2024/2024-ssl-el-robot-placement.adoc]: I believe that `Start` and `Goal` are subtopics of `scenarios`, right?
I think they both should be at a higher title level. ie
```
=== Scenarios.
==== Start
==== Goal
```

File: 2024/2024-ssl-el-robot-placement.adoc [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+image::scenarios_robot_placement/free_kick1_start.svg[Interactive,700,opts=interactive]
+== Goal
+image::scenarios_robot_placement/free_kick1_goal.svg[Interactive,700,opts=interactive]
+
+=== Scenario 2: Kick Off
+== Start
```
Comment (ersaraujo) [inline 2024/2024-ssl-el-robot-placement.adoc]: I believe that `Start` and `Goal` are subtopics of `scenarios`, right?
I think they both should be at a higher title level. ie
```
=== Scenarios.
==== Start
==== Goal
```

File: 2024/2024-ssl-el-robot-placement.adoc [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+image::scenarios_robot_placement/kick_off_start.svg[Interactive,700,opts=interactive]
+== Goal
+image::scenarios_robot_placement/kick_off_goal.svg[Interactive,700,opts=interactive]
+
+=== Scenario 3: Free Kick 2
+== Start
```
Comment (ersaraujo) [inline 2024/2024-ssl-el-robot-placement.adoc]: I believe that `Start` and `Goal` are subtopics of `scenarios`, right?
I think they both should be at a higher title level. ie
```
=== Scenarios.
==== Start
==== Goal
```

File: 2024/2024-ssl-el-robot-placement.adoc [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+image::scenarios_robot_placement/free_kick2_start.svg[Interactive,700,opts=interactive]
+== Goal
+image::scenarios_robot_placement/free_kick2_goal.svg[Interactive,700,opts=interactive]
+
+Scenario 4: Penalty
+== Start
```
Comment (ersaraujo) [inline 2024/2024-ssl-el-robot-placement.adoc]: I believe that `Start` and `Goal` are subtopics of `scenarios`, right?
I think they both should be at a higher title level. ie
```
=== Scenarios.
==== Start
==== Goal
```

**Review by ersaraujo — CHANGES_REQUESTED** [review]

**Review by ersaraujo — APPROVED** [review]

**Review by JoaoVictor0508 — APPROVED** [review]

**Pull request merged**